src/test/dafny/proofs/FM-paper.dfy

```diff
@@ -161,7 +161,7 @@ module Kontract1 {
      *              The stack content is unconstrained but there must be
      *              enough capacity (3) to perform this computation.
      */
-    method Loopy(st: ExecutingState, c: u8) returns (st': State)
+    method {:verify false} Loopy(st: ExecutingState, c: u8) returns (st': State)
```

Comment:
Isn't this risky in a file related to a paper?

Reply:
I meant that if someone comes looking to check out the paper and finds that the code is not verified they might get the wrong impression. 
Given that the paper has a concrete publication date maybe it'd be best to have a tag or branch so that it's clear what was the paper's state. Ideally the tag would also be mentioned in the paper.

Reply:
Well, we can do that ... but I personally wouldn't worry too much about it.  If *anyone* actually looks at this and notices what's going on ... then I will be super duper happy.  Don't hold your breath on that happening though.  I'd give it a probability of something very small like ... 0.0000001.